pkg/notifiers/format.go

```diff
@@ -204,6 +208,16 @@ func FormatNetworkPolicyYAML(yaml string, clusterName string, funcMap template.F
 	return tpl.String(), nil
 }
 
+// ptrTypeIs checks if src is of type target. If src is a pointer or interface, it looks at its actual type
+// sprig has a `typeIs` and `kindIs` but it doesn't work cleanly with pointers
+func ptrTypeIs(target string, src interface{}) bool {
```

Comment:
This function can find more value if you move it `pkg/pointers.go`. Just make a note that it is non-recursive, in that, it does not go down to leaf nodes of the AST.

Reply:
The fact that it is non-recursive and not fully tested for all sorts of pointer/iface types is why I avoided making it generic. For now I'll just make a specific function as @fredrb suggested